examples/sample-pipeline/README.md

```diff
@@ -43,41 +27,36 @@ You can verify these definitions using [tapstry-pipelines](https://github.com/ta
 % tapestry-pipelines tkn verify -d . -i icr.io/gitsecure -t v1 -key ssf-verify.pub
 ```
 
-## Setup Pipeline
+## Starting Demo
 
 ```bash
-# Setup access secrets
-% kubectl create -f registry-secret.yaml
-% kubectl create -f pipeline-account.yaml
-
-# Setup core tasks
-% kubectl create -f task-git-clone.yaml
-% kubectl create -f task-build-push-image.yaml
-% kubectl create -f task-deploy.yaml
-% kubectl create -f task-syft-bom-generation.yaml
-% kubectl create -f task-gyrpe-scan.yaml
-
-# Setup pipeline
-% kubectl create -f ssf-pipeline.yaml
-```
+# Only if a cluster is needed.
+make setup-minikube
 
-## Run Pipeline
+# Setup tekton w/ chains
+make setup-tekton-chains tekton-generate-keys setup-kyverno
 
-In this example, we are using [tekton-tutorial-openshift](https://github.com/IBM/tekton-tutorial-openshift) application sample application. If you want to use other application, make sure to make corresponding changes to `ssf-run.yaml`. You may also have to change params to `build` and `deploy` tasks to reflect correct location for `Dockerfile` context and deployment file resp.
+# Run a new pipeline.
+make example-sample-pipeline
 
-```bash
-% kubectl create -f ssf-run.yaml
-```
+# Export the value of APP_IMAGE from the pipelinerun describe as DOCKER_IMG:
+export DOCKER_IMG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="APP_IMAGE")].value}')
```

Comment:
I don't think this is quite right. I think it's more like what is in `ibm-tutorials`. e.g. 
```bash
$ tkn pr describe --last -o jsonpath='{.spec.params}' | jq
[
  {
    "name": "gitUrl",
    "value": "https://github.com/IBM/tekton-tutorial-openshift"
  },
  {
    "name": "pathToYamlFile",
    "value": "kubernetes/picalc.yaml"
  },
  {
    "name": "imageUrl",
    "value": "192.168.1.37:8888/hello-ssf"
  },
  {
    "name": "imageTag",
    "value": "1h"
  }
]
```
Doesn't contain `APP_IMAGE`.

Reply:
oops. Fixed to use `imageUrl`